packages/logs/README.md

```diff
@@ -219,12 +219,84 @@ The Datadog backend adds more fields, like:
 - `http.useragent`
 - `network.client.ip`
 
-### Status parameter
+### Error tracking
 
-After the Datadog browser logs SDK is initialized, send a custom log entry to Datadog with the API using the status as a parameter:
+The Datadog browser logs SDK allows for manual error tracking by using the optional `error` parameter. When an instance of a [JavaScript Error][10] is provided, the SDK will extract relevant information (kind, message, stack trace) from the error.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to indicate from which browser sdk version that feature is available.
note: in any case, we need to wait for the previous PR to be released to merge this one.

Reply:
👌 4.36.0